### PR TITLE
Runs push-pods after make-release

### PR DESCRIPTION
**Pull request by vegaro**
Pushes the pod in its own job until https://github.com/CocoaPods/CocoaPods/issues/11621 is fixed. Otherwise make-release always fails

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -724,8 +734,14 @@ workflows:
             - hold
           <<: *release-branches
       - make-release:
+          xcode_version: '14.1.0'
+          <<: *release-tags
+      - push-pods:
           # Xcode 14 not supported until https://github.com/CocoaPods/CocoaPods/issues/11558 is fixed.
+          # This runs as its own job until https://github.com/CocoaPods/CocoaPods/issues/11621 is fixed.
```
Comment (NachoSoto) [inline .circleci/config.yml:741]: <3

File: fastlane/Fastfile [hunk 2]
```diff
@@ -237,15 +237,22 @@ platform :ios do
...
+    # making it its own lane until https://github.com/CocoaPods/CocoaPods/issues/11621 is fixed.
+    # push_pods 
     carthage_archive
     export_xcframework
     github_release(version: version_number)
   end
 
+  desc "Release to CocoaPods"
+  lane :push_pods do |options|
+    version_number = current_version_number
```
Comment (NachoSoto) [inline fastlane/Fastfile:252]: Is this used?

File: .circleci/config.yml [hunk 3]
```diff
@@ -724,8 +734,14 @@ workflows:
             - hold
           <<: *release-branches
       - make-release:
+          xcode_version: '14.1.0'
```
Comment (vegaro) [inline .circleci/config.yml:737]: I updated xcode here @NachoSoto

**Review by NachoSoto — APPROVED** [review]
Awesome 🎉

File: fastlane/Fastfile (hunk 2 above)
Comment (vegaro) [reply fastlane/Fastfile:252]: Oh no it's not. I'll remove it

**Pull request merged**